builder/vmware/common/step_compact_disk.go

```diff
@@ -26,28 +28,46 @@ type StepCompactDisk struct {
 func (s StepCompactDisk) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
 	driver := state.Get("driver").(Driver)
 	ui := state.Get("ui").(packer.Ui)
-	full_disk_path := state.Get("full_disk_path").(string)
+	diskPaths := state.Get("diskFullPaths").([]string)
```

Comment:
In general the convention I've seen is that we tend to store things in state with underscored names rather than camelcase

Reply:
... happy to stick with convention though! I'll change this to `disk_full_paths`